dev/prod_builders.json

```diff
@@ -1149,7 +1149,7 @@
     {
       "name": "Mac_ios platform_channels_benchmarks",
```

Comment:
With required update in infra config side, here may need update as well.

Reply:
done